python/nammu/controller/MenuController.py

```diff
@@ -36,6 +35,9 @@ def saveFile(self):
     def closeFile(self):
         self.controller.closeFile()
     
+    def printFile(self):
```

Comment:
Agree. Probably none of the intermediate controllers should exist, except for perhaps the edit/model area controller. I think I created some issues about changing this, but haven't done it yet. Design in general needs some simplification.